darts/timeseries.py

```diff
@@ -41,25 +42,38 @@ def __init__(self,
         fill_missing_dates
             Optionally, a boolean value indicating whether to fill missing dates with NaN values
             in case the frequency of `series` cannot be inferred.
+        dummy_index
+            Optionally, if no date time index is present or even available. Creates a dummy index to be
```

Comment:
I would rephrase: "Optionally, if no date time index is present, this flag will instruct Darts to build a dummy time index in order to obtain a valid TimeSeries. This option can be used in cases where the time index doesn't matter.".

Reply:
Done